### PR TITLE
#44: few dashboard fixes + LinkSpeed zero when LinkStatus is zero

### PR DESCRIPTION
**Pull request by RaymondMouthaan**
Fixes a few settings in the Dashboard and sets LinkSpeed to zero when LinkStatus is zero.

see #44

### CODE REVIEW

File: pkg/omada/handler.go [hunk 1]
```diff
@@ -1,126 +1,126 @@
-package omada
```
Comment (RaymondMouthaan) [inline pkg/omada/handler.go:1]: apologise for the code formatting

File: pkg/omada/handler.go [hunk 2]
```diff
@@ -1,126 +1,126 @@
...
+			omada_device_poe_remain_watts.WithLabelValues(item.Name, item.Model, item.Version, item.Ip, item.Mac, site, siteId, item.Type).Set(item.PoeRemain)
+		}
+	}
+}
+
+// set prometheus metrics for ports
+func setPortMetricsByDevice(c *api.Client, device api.Device, site string, siteId string) error {
+	for _, p := range device.Ports {
+		linkSpeed := float64(0)
+		if p.PortStatus.LinkStatus == 0 || p.PortStatus.LinkSpeed == 0 {
```
Comment (RaymondMouthaan) [inline pkg/omada/handler.go:74]: this is the line that matters

File: pkg/omada/handler.go (hunk 1 above)
Comment (charlie-haley) [reply pkg/omada/handler.go:1]: Could you please reformat this file with a single tab indentation just so it matches the rest of the project? Happy to do this for you if you need

**Review by charlie-haley — APPROVED** [review]
LGTM

**Pull request merged**